addons/sale/views/variant_templates.xml

```diff
@@ -58,7 +58,7 @@
                                         <div>
                                             <input type="radio"
                                                 t-attf-class="js_variant_change radio_input #{variant_id.attribute_id.create_variant}"
-                                                t-att-checked="value_id in combination"
+                                                t-att-checked="value_id in combination or single_and_custom"
```

Comment:
Hello.

I would have to dive back into that code to give you more insights but I'm unsure how this fixes an issue, even after reading your commit message. Could you elaborate?
Also, does this only apply to the "radio" type? The "select" and "color" types have similar code.

Reply:
you are right it would also apply for `color` and maybe `select` since the issue is that nothing is selected and the field is not shown because of:

```xml
t-attf-class="list-inline  #{'d-none' if single_and_custom else ''}">
```

I expanded on the issue on a direct message.